libs/domain/content/link/link.component.ts

```diff
@@ -22,10 +36,15 @@ export class ContentLinkComponent extends ContentMixin<
   });
 
   protected override render(): TemplateResult | void {
-    const { button, icon, singleLine, color } = this.$options();
+    const { appearance, button, icon, singleLine, color } = this.$options();
+
+    const dropdown = appearance === ContentLinkAppearance.Dropdown;
 
-    if (button) {
-      return html`<oryx-button>${this.renderLink(true)}</oryx-button>`;
+    if (button || dropdown || appearance === ContentLinkAppearance.Button) {
+      return html`<oryx-button
+        .type=${ifDefined(dropdown ? ButtonType.Text : undefined)}
+        >${this.renderLink(true)}</oryx-button
+      >`;
```

Comment:
Our initial designs started to use buttons a lot, but in reality those are links.
We like the UI/UX to start to use native links and we already agreed with Helen on this approach. 

Can we already start using `<oryx-link>` here, and apply the navigation styles to it?